definitions/divorce/json/CaseEvent/CaseEvent-bailiff-nonprod.json

```diff
@@ -60,9 +60,10 @@
     "Description": "Draft Acknowledgement of Service",
     "DisplayOrder": 1,
     "PreConditionState(s)": "AosAwaitingSol;AosAwaiting;AwaitingBailiffReferral;AwaitingBailiffService;IssuedToBailiff",
```

Comment:
I think to be able to run this event from the new states with a solicitor, you will need to add "CRU" permissions to the solicitor on each of those new states. Might be worth checking.

Reply:
Permissions already there 👍